OneSchedule.Domain/NotificationScheduler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OneSchedule.Domain.Abstractions;
+using OneSchedule.Entities;
+using Quartz;
+using Quartz.Impl;
+using Quartz.Lambda;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Telegram.Bot;
+
+namespace OneSchedule.Domain
+{
+    public class NotificationScheduler : INotificationScheduler
+    {
+        private readonly ITelegramBotClient _bot;
+
+        public NotificationScheduler(ITelegramBotClient bot)
+        {
+            _bot = bot;
+        }
+
+        public async Task ScheduleNotifications(EventEntity eventEntity)
+        {
+            IScheduler scheduler = SchedulerRepository.Instance.Lookup("Quartz ASP.NET�Core Sample Scheduler").Result;
```

Comment:
Why do you use Result property. Lookup() is asynchronous operation and you have to use await keyword